Google Colab resource: Rewrite to emphasize GPU computation

swfarnsworth opened the pull request:
Previously, it was unclear why this was a data science resource. This version states that Colab offers GPU computation and that it can be used for deep learning, which is the reason. Also fixes abrupt change in sentence structure ("a ... Jupyter Notebook which supports e.g. !apt-get"; "e.g." unexpected).

swfarnsworth committed:
Rewrite to emphasize GPU computation

Previously, it was unclear why this was a data science resource. This version states that Colab offers GPU computation and that it can be used for deep learning, which is the reason. Also fixes abrupt change in sentence structure ("a ... Jupyter Notebook which supports e.g. !apt-get"; "e.g." unexpected).

ChrisLovering commented on pydis_site/apps/resources/resources/google_colab.yaml:
```diff
@@ -1,7 +1,7 @@
-  and well-integrated with Google Drive.
-  You can share your Colab Notebooks with other people and work collaboratively.
+description: Google Colab is a custom version of Jupyter Notebook that runs code in the cloud, with support for 
+  Debian package installation with `apt-get`. You can share your Colab Notebooks with other people and work collaboratively. 
```
Actually, just noticed this \`apt-get` won't format it as code, not sure if we have another way to do this.

swfarnsworth replied:
We can just delete that part and say "support for Debian packages".

swfarnsworth committed:
linting: remove trailing spaces

swfarnsworth committed:
Remove mention of Debian packages

Expressions in backticks (apt-get) won't be rendered as code, though since Colab is mostly a beginner sandbox, I don't think the word "Debian" will often be known.
